src/Command/ImportExisting.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\Takedown\Takedown;
+use App\Entity\Takedown\ChildProtection\ChildProtection;
+use App\Entity\Takedown\Dmca\Dmca;
+use App\Util\TakedownUtilInterface;
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\DBAL\Connection;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ImportExisting extends Command {
+
+	/**
+	 * @var Registry
+	 */
+	protected $doctrine;
+
+	/**
+	 * @var Connection
+	 */
+	protected $old;
+
+	/**
+	 * @var TakedownUtilInterface
+	 */
+	protected $takedownUtil;
+
+	/**
+	 * @var string[]
+	 */
+	protected const SITE_IDS = [
+		'en.wikipedia.org' => 'enwiki',
+		'wikimediafoundation.org' => 'foundationwiki',
+		'cy.wikipedia.org' => 'cywiki',
+		'en.wiktionary.org' => 'enwiktionary',
+		'commons.wikimedia.org' => 'commonswiki',
+	];
+
+	/**
+	 * @var string[]
+	 */
+	protected const PROJECT_IDS = [
+		'Wikimedia Commons' => 'commonswiki',
+	];
+
+	/**
+	 * @var string[]
+	 */
+	protected const USER_NAMES = [
+		'kbrown' => 'Kbrown (WMF)',
+		'ktsouroupidou' => 'Kalliope (WMF)',
+	];
+
+	/**
+	 * @var string[]
+	 */
+	protected const APPROVER_NAMES = [
+		'Geoff Brigham' => 'GeoffBrigham (WMF)',
```

Comment:
Geoff doesn't work for us anymore.

Reply:
@MaxSem I figured, but his reference is still in the old database. :/